src/main/groovy/jenkins/automation/utils/CheckmarxSecurityJobBuilder.groovy

```diff
@@ -0,0 +1,63 @@
+package jenkins.automation.utils
+
+import javaposse.jobdsl.dsl.DslFactory
+import javaposse.jobdsl.dsl.Job
+
+
+class CheckmarxSecurityJobBuilder {
+
+    String name
+    String description
+    String baseUrl
+    String scanRepo
+    String checkmarxComment
+    Boolean vulnerabilityThresholdEnabled
+    String highThreshold
+    String mediumThreshold
+    String lowThreshold
+
+    Job build(DslFactory factory){
+        factory.job(name){
+            it.description this.description
+            
+            scm {
+                git {
```

Comment:
seems to me we should probably support multiple repos, similar to how the base job builder does it. In fact, I wonder: shouldn't this whole thing use BaseJobBuilder